src/event.rs

```diff
@@ -640,14 +640,14 @@ pub enum MouseScrollDelta {
     ///
     /// Positive values indicate movement forward
     /// (away from the user) or rightwards.
-    LineDelta(f32, f32),
+    LineDelta(PhysicalPosition<f64>),
```

Comment:
I don't think `LineDelta` should use the position type. `PhysicalPosition` refers to pixels, but lines aren't pixels.

Reply:
I think that makes a lot of sense, yeah. A simple f32 tuple the way it was before should be good.